widget/embedded/src/pages/ConfirmSwapPage.tsx

```diff
@@ -291,7 +291,7 @@ export function ConfirmSwapPage() {
           setShowQuoteWarningModal(false);
           await addNewSwap();
         }}
-        onChangeSettings={() => navigate(navigationRoutes.settings)}
+        onChangeSettings={() => navigate(`/${navigationRoutes.settings}`)}
```

Comment:
Is this change necessary?

Reply:
Yes, it was necessary to alter because the page address wasn't correct when we tried to access the settings from this page.
This is how it appeared:
http://localhost:3002/confirm-swap/settings

Reply:
I agree. Is it possible for you to utilize relative paths as done in other locations ?
confirm-swap page :
![1](https://github.com/rango-exchange/rango-client/assets/120931880/b3568b47-1388-441e-a955-b141fac5733c)

Please review other comments related to navigation as well

Reply:
yes, It's fixed